Resources/doc/reference/api.rst

```diff
@@ -32,6 +32,12 @@ Here's the configuration we used, you may adapt it to your needs:
 
     twig:
         exception_controller: 'FOS\RestBundle\Controller\ExceptionController::showAction'
+    
+    # for SonataPageBundle > 2.3.6    
```

Comment:
Note sure if it should be repeated on this part.

I'm not the best person to decide about it.

Let's @rande of another page-bundle collaborator review this PR.